examples/framework/preferences/minimalFootprint/fullPage.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/normalize/css/normalize.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/framework/core/css/fluid.css" />
+
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/jquery/ui/css/fl-theme-by/by.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/jquery/ui/css/fl-theme-yb/yb.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/jquery/ui/css/fl-theme-bw/bw.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/jquery/ui/css/fl-theme-wb/wb.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/lib/jquery/ui/css/fl-theme-lgdg/lgdg.css" />
+
+        <!-- Component styles -->
+        <link rel="stylesheet" type="text/css" href="../../../../src/framework/preferences/css/Enactors.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/framework/preferences/css/PrefsEditor.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/framework/preferences/css/FullPrefsEditor.css" />
+        <link rel="stylesheet" type="text/css" href="../../../../src/framework/preferences/css/FullPreviewPrefsEditor.css" />
+
+        <!-- Demo style -->
+        <link rel="stylesheet" type="text/css" href="css/prefsEditorFullPage.css" />
+
+        <title>Preferences Editor: Full Page</title>
+        <script type="text/javascript" src="../../../../src/lib/jquery/core/js/jquery.js"></script>
```

Comment:
Similarly a comment here would be very helpful listing the path of the built file that all these includes could be replaced with in the "dist" directory if it were there